api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -184,6 +194,24 @@ public void createRuntime(Runtime runtime, String workspaceFirecloudName) {
         });
   }
 
+  @Override
+  public void updateRuntime(String workspaceNamespace, UpdateRuntimeRequest updateRuntimeRequest) {
+    leonardoRetryHandler.run(
+        (context) -> {
+          runtimesApiProvider
+              .get()
+              .updateRuntime(
+                  updateRuntimeRequest.getRuntime().getGoogleProject(),
+                  updateRuntimeRequest.getRuntime().getRuntimeName(),
+                  new LeonardoUpdateRuntimeRequest()
```

Comment:
Please add a TODO somewhere around here and/or file ticket as a follow-up for updating labels, Leo ticket is https://broadworkbench.atlassian.net/browse/IA-2149